public_dropin_environments/python3_keras/requirements.txt

```diff
@@ -1,6 +1,6 @@
 tensorflow==2.2.1
 Keras==2.3.1
-numpy==1.19.0
+numpy<1.19.0
```

Comment:
this is not a correct rule, as far as I see the rule is `numpy<1.19.0 and >=1.16.0`

Current rule would be fine with numpy 0.1 technically.

Reply:
```
numpy>=1.16.0,<1.19.0
```

Reply:
I suppose it will install 1.18.5 anyhow, but I'll change

Reply:
**DONE**